src/lib.rs

```diff
@@ -126,6 +126,6 @@ pub use waker::Waker;
 #[cfg(unix)]
 pub mod unix {
     //! Unix only extensions.
-
+    pub use crate::sys::SocketAddr;
```

Comment:
I'm not too happy this, although I completely understand. Has there been any effort to add a way to create a `SocketAddr` to std lib?

Reply:
We probably should open an issue. If we match the API of `std`, it should not be a breaking change to switch this export to be the std version once we can use it.

Reply:
Sure I can open an issue. I don't see any reason that would not be okay? It should be pretty easy to follow up with the fix for it as well.